WDOTS/wdots.github.io
# Check for changed dependencies

# What has changed?

Added a post-merge hook that runs after the developer runs `git pull`. This checks `package.json` for changes. If there have been changes, a warning is displayed (using [megalog](https://github.com/sndrs/megalog)) that nudges the developer to run `npm install` before continuing development.

# How does it help?

✨ Ensures that the developer always has the correct dependencies installed when working on the project. ✨

# Screenshot

<img width="838" alt="screen shot 2016-12-28 at 11 37 39" src="https://cloud.githubusercontent.com/assets/5931528/21518906/2db0e3b4-ccf2-11e6-92ef-1008cc1e3448.png">

# Steps to test

1. Pull down this branch. Run `npm install` to install the new git hook and `megalog`
2. Create a new branch from this one (`git checkout -b test-messaging-tool`)
3. Install an npm module and save to package.json (`npm install left-pad --save`)
4. Add and commit change (`git commit -am 'test'`)
5. Push branch to GitHub (`git push -u origin test-messaging-tool`)
6. Reset hard to previous commit (`git reset --hard HEAD~1`)
7. Pull down latest (`git pull`)
8. You should see a warning, as presented in the screenshot above.
